File: app/src/main/java/com/hedgehog/gdzietabiedra/pojo/Shops/Shop.java

package com.hedgehog.gdzietabiedra.pojo.Shops;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import io.realm.RealmObject;
import io.realm.annotations.*;

public class Shop extends RealmObject{

    @Expose
    @PrimaryKey
    private String id;
    @Expose
    private String name;
    @Expose
    private String city;
    @SerializedName("city_slug")
    @Expose
    private String citySlug;
    @Expose
    private String street;
    @SerializedName("street_number")
    @Expose
    private String streetNumber;
    @Expose
    private String latitude;
    @Expose
    private String longitude;
    @Expose
    private String hours;
    @SerializedName("hours_saturday")
    @Expose
    private String hoursSaturday;
    @SerializedName("hours_sunday")
    @Expose
    private String hoursSunday;
    @Expose
    private String bakery;
    @Expose
    private String relax;
    @Expose
    private String atm;
    @SerializedName("card_payment")
    @Expose
    private String cardPayment;
//    @SerializedName("_new")
//    @Expose
//    private String _new;
    @Expose
    private String special;
    @Expose
    private String sublease;
    @SerializedName("shop_number")
    @Expose
    private String shopNumber;
    @SerializedName("import_query")
    @Expose
    private String importQuery;
    @SerializedName("import_update")
    @Expose
    private String importUpdate;
    @Expose
    private String distance;

    /**
     * 
     * @return
     *     The id
     */
    public String getId() {
        return id;
    }

    /**
     * 
     * @param id
     *     The id
     */
    public void setId(String id) {
        this.id = id;
    }

    /**
     * 
     * @return
     *     The name
     */
    public String getName() {
        return name;
    }

    /**
     * 
     * @param name
     *     The name
     */
    public void setName(String name) {
        this.name = name;
    }

    /**
     * 
     * @return
     *     The city
     */
    public String getCity() {
        return city;
    }

    /**
     * 
     * @param city
     *     The city
     */
    public void setCity(String city) {
        this.city = city;
    }

    /**
     * 
     * @return
     *     The citySlug
     */
    public String getCitySlug() {
        return citySlug;
    }

    /**
     * 
     * @param citySlug
     *     The city_slug
     */
    public void setCitySlug(String citySlug) {
        this.citySlug = citySlug;
    }

    /**
     * 
     * @return
     *     The street
     */
    public String getStreet() {
        return street;
    }

    /**
     * 
     * @param street
     *     The street
     */
    public void setStreet(String street) {
        this.street = street;
    }

    /**
     * 
     * @return
     *     The streetNumber
     */
    public String getStreetNumber() {
        return streetNumber;
    }

    /**
     * 
     * @param streetNumber
     *     The street_number
     */
    public void setStreetNumber(String streetNumber) {
        this.streetNumber = streetNumber;
    }

    /**
     * 
     * @return
     *     The latitude
     */
    public String getLatitude() {
        return latitude;
    }

    /**
     * 
     * @param latitude
     *     The latitude
     */
    public void setLatitude(String latitude) {
        this.latitude = latitude;
    }

    /**
     * 
     * @return
     *     The longitude
     */
    public String getLongitude() {
        return longitude;
    }

    /**
     * 
     * @param longitude
     *     The longitude
     */
    public void setLongitude(String longitude) {
        this.longitude = longitude;
    }

    /**
     * 
     * @return
     *     The hours
     */
    public String getHours() {
        return hours;
    }

    /**
     * 
     * @param hours
     *     The hours
     */
    public void setHours(String hours) {
        this.hours = hours;
    }

    /**
     * 
     * @return
     *     The hoursSaturday
     */
    public String getHoursSaturday() {
        return hoursSaturday;
    }

    /**
     * 
     * @param hoursSaturday
     *     The hours_saturday
     */
    public void setHoursSaturday(String hoursSaturday) {
        this.hoursSaturday = hoursSaturday;
    }

    /**
     * 
     * @return
     *     The hoursSunday
     */
    public String getHoursSunday() {
        return hoursSunday;
    }

    /**
     * 
     * @param hoursSunday
     *     The hours_sunday
     */
    public void setHoursSunday(String hoursSunday) {
        this.hoursSunday = hoursSunday;
    }

    /**
     * 
     * @return
     *     The bakery
     */
    public String getBakery() {
        return bakery;
    }

    /**
     * 
     * @param bakery
     *     The bakery
     */
    public void setBakery(String bakery) {
        this.bakery = bakery;
    }

    /**
     * 
     * @return
     *     The relax
     */
    public String getRelax() {
        return relax;
    }

    /**
     * 
     * @param relax
     *     The relax
     */
    public void setRelax(String relax) {
        this.relax = relax;
    }

    /**
     * 
     * @return
     *     The atm
     */
    public String getAtm() {
        return atm;
    }

    /**
     * 
     * @param atm
     *     The atm
     */
    public void setAtm(String atm) {
        this.atm = atm;
    }

    /**
     * 
     * @return
     *     The cardPayment
     */
    public String getCardPayment() {
        return cardPayment;
    }

    /**
     * 
     * @param cardPayment
     *     The card_payment
     */
    public void setCardPayment(String cardPayment) {
        this.cardPayment = cardPayment;
    }

    /**
     * 
     * @return
     *     The _new
     */
//    public String getNew() {
//        return _new;
//    }
//
//    /**
//     *
//     * @param _new
//     *     The new
//     */
//    public void setNew(String _new) {
//        this._new = _new;
//    }

    /**
     * 
     * @return
     *     The special
     */
    public String getSpecial() {
        return special;
    }

    /**
     * 
     * @param special
     *     The special
     */
    public void setSpecial(String special) {
        this.special = special;
    }

    /**
     * 
     * @return
     *     The sublease
     */
    public String getSublease() {
        return sublease;
    }

    /**
     * 
     * @param sublease
     *     The sublease
     */
    public void setSublease(String sublease) {
        this.sublease = sublease;
    }

    /**
     * 
     * @return
     *     The shopNumber
     */
    public String getShopNumber() {
        return shopNumber;
    }

    /**
     * 
     * @param shopNumber
     *     The shop_number
     */
    public void setShopNumber(String shopNumber) {
        this.shopNumber = shopNumber;
    }

    /**
     * 
     * @return
     *     The importQuery
     */
    public String getImportQuery() {
        return importQuery;
    }

    /**
     * 
     * @param importQuery
     *     The import_query
     */
    public void setImportQuery(String importQuery) {
        this.importQuery = importQuery;
    }

    /**
     * 
     * @return
     *     The importUpdate
     */
    public String getImportUpdate() {
        return importUpdate;
    }

    /**
     * 
     * @param importUpdate
     *     The import_update
     */
    public void setImportUpdate(String importUpdate) {
        this.importUpdate = importUpdate;
    }

    /**
     * 
     * @return
     *     The distance
     */
    public String getDistance() {
        return distance;
    }

    /**
     * 
     * @param distance
     *     The distance
     */
    public void setDistance(String distance) {
        this.distance = distance;
    }

}

File: app/src/main/java/com/hedgehog/gdzietabiedra/pojo/Shops/Shop.kt
package com.hedgehog.gdzietabiedra.pojo.Shops

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class Shop : RealmObject() {

    /**

     * @return
     * *     The id
     */
    /**

     * @param id
     * *     The id
     */
    @Expose
    @PrimaryKey
    var id: String? = null
    /**

     * @return
     * *     The name
     */
    /**

     * @param name
     * *     The name
     */
    @Expose
    var name: String? = null
    /**

     * @return
     * *     The city
     */
    /**

     * @param city
     * *     The city
     */
    @Expose
    var city: String? = null
    /**

     * @return
     * *     The citySlug
     */
    /**

     * @param citySlug
     * *     The city_slug
     */
    @SerializedName("city_slug")
    @Expose
    var citySlug: String? = null
    /**

     * @return
     * *     The street
     */
    /**

     * @param street
     * *     The street
     */
    @Expose
    var street: String? = null
    /**

     * @return
     * *     The streetNumber
     */
    /**

     * @param streetNumber
     * *     The street_number
     */
    @SerializedName("street_number")
    @Expose
    var streetNumber: String? = null
    /**

     * @return
     * *     The latitude
     */
    /**

     * @param latitude
     * *     The latitude
     */
    @Expose
    var latitude: String? = null
    /**

     * @return
     * *     The longitude
     */
    /**

     * @param longitude
     * *     The longitude
     */
    @Expose
    var longitude: String? = null
    /**

     * @return
     * *     The hours
     */
    /**

     * @param hours
     * *     The hours
     */
    @Expose
    var hours: String? = null
    /**

     * @return
     * *     The hoursSaturday
     */
    /**

     * @param hoursSaturday
     * *     The hours_saturday
     */
    @SerializedName("hours_saturday")
    @Expose
    var hoursSaturday: String? = null
    /**

     * @return
     * *     The hoursSunday
     */
    /**

     * @param hoursSunday
     * *     The hours_sunday
     */
    @SerializedName("hours_sunday")
    @Expose
    var hoursSunday: String? = null
    /**

     * @return
     * *     The bakery
     */
    /**

     * @param bakery
     * *     The bakery
     */
    @Expose
    var bakery: String? = null
    /**

     * @return
     * *     The relax
     */
    /**

     * @param relax
     * *     The relax
     */
    @Expose
    var relax: String? = null
    /**

     * @return
     * *     The atm
     */
    /**

     * @param atm
     * *     The atm
     */
    @Expose
    var atm: String? = null
    /**

     * @return
     * *     The cardPayment
     */
    /**

     * @param cardPayment
     * *     The card_payment
     */
    @SerializedName("card_payment")
    @Expose
    var cardPayment: String? = null
    //    @SerializedName("_new")
    //    @Expose
    //    private String _new;
    /**

     * @return
     * *     The _new
     */
    //    public String getNew() {
    //        return _new;
    //    }
    //
    //    /**
    //     *
    //     * @param _new
    //     *     The new
    //     */
    //    public void setNew(String _new) {
    //        this._new = _new;
    //    }

    /**

     * @return
     * *     The special
     */
    /**

     * @param special
     * *     The special
     */
    @Expose
    var special: String? = null
    /**

     * @return
     * *     The sublease
     */
    /**

     * @param sublease
     * *     The sublease
     */
    @Expose
    var sublease: String? = null
    /**

     * @return
     * *     The shopNumber
     */
    /**

     * @param shopNumber
     * *     The shop_number
     */
    @SerializedName("shop_number")
    @Expose
    var shopNumber: String? = null
    /**

     * @return
     * *     The importQuery
     */
    /**

     * @param importQuery
     * *     The import_query
     */
    @SerializedName("import_query")
    @Expose
    var importQuery: String? = null
    /**

     * @return
     * *     The importUpdate
     */
    /**

     * @param importUpdate
     * *     The import_update
     */
    @SerializedName("import_update")
    @Expose
    var importUpdate: String? = null
    /**

     * @return
     * *     The distance
     */
    /**

     * @param distance
     * *     The distance
     */
    @Expose
    var distance: String? = null

}
